aspnetcore/host-and-deploy/health-checks.md

```diff
@@ -1831,4 +1831,24 @@ app.UseEndpoints(endpoints =>
 
 For more information, see <xref:fundamentals/middleware/index#branch-the-middleware-pipeline>.
 
+## Dependency Injection and Health Checks
+
+It is possible to use dependency injection to consume an instance of a specific Type inside a Health Check class. This can be useful to inject options or a global configuration to a Health Check.
+
+> Using dependency injection is not a common scenario to configure Health Checks. Usually, each Health Check is quite specific to the actual test and is configured using `IHealthChecksBuilder` extension methods.
+
+The following example shows a sample Health Check that retrieves a configuration object via dependency injection:
+
+:::code language="csharp" source="health-checks/samples/7.x/HealthChecksSample/HealthChecks/SampleHealthCheckWithDI.cs" id="snippet_Class":::
+
+The used `SampleHealthCheckWithDiConfig` needs to be added to the service container as well as the actual Health Check:
```

Comment:
```suggestion
The `SampleHealthCheckWithDiConfig` and the Health check needs to be added to the service container :
```

Reply:
@JuergenGutsch  I can fix this if you like. Let me know.

Reply:
Thank you. It would be cool, if you would fix this.